cobc/ChangeLog

```diff
@@ -1,4 +1,14 @@
 
+2022-07-28  Nicolas Berthier <nicolas.berthier@ocamlpro.com>
+
+	* flag.def, config.def: make format a dialect option instead of a flag
+	* help.c: mention alias for options --free and --fixed
+	* cobc.h: extend cb_format_enum with COBOL85 (fixed format with
+	  pending Area A enforcement); define CB_SF_FIXED and CB_SF_FREE
+	* cobc.c: use new source format macros, whitespace fixes
+	* parser.y, pplex.l: annot every token that is not used in the grammar,
```

Comment:
/annot/annotate/ - no use in mentioning the whitespace fixes (not relevant for copyright or future programmers)